### ISSUE TITLE
Terminal: The terminal gnome-terminal was not found

### ISSUE THREAD

**Issue by WillisCat** [wbond/sublime_terminal#159]
I use the Elementary os ,how name is the terminal?

**Comment by twolfson** [comment]
Our current list of terminal mappings can be found here:

https://github.com/wbond/sublime_terminal/blob/1.16.2/Terminal.py#L98-L110

I'm not sure which terminal Elementary OS's default is but if you want to specify a terminal directly, that can be done via our settings as documented here:

https://github.com/wbond/sublime_terminal/tree/1.16.2#package-settings

**Comment by Poutas** [comment]
Why close this issue ? 
I via google find the elementary os terminal name is "pantheon-terminal".
Now the plugin is normal.
This is the url [https://www.reddit.com/r/elementaryos/comments/36dsc5/what_is_the_name_of_the_terminal_program_in_freya/](url)

**Comment by twolfson** [comment]
Sorry, I misunderstood the initial question. I thought you were asking about how to specify `gnome-terminal` or how to override resolution of `gnome-terminal`.

Would you be interested in opening a PR to fix the behavior for Elementary OS so it opens `pantheon-terminal` by default?

**Comment by Poutas** [comment]
I am sorry,I not very familiar with python and linux shell.

**Comment by twolfson** [comment]
Okay, then could you provide us with the following knowledge:

- What is the output of `ps ax | grep sess`?
- What happens when you run `pantheon-terminal`?

**Comment by Poutas** [comment]
I am sorry I can't  provide the message now.

**Comment by twolfson** [comment]
Can you elaborate by you can't provide the message?

**Comment by Poutas** [comment]
Is this ?
![屏幕截图 2017-02-02 10:17:10.png](https://ooo.0o0.ooo/2017/02/02/5892974565bcd.png)

**Comment by twolfson** [comment]
Yep, that's what we needed. I will write a patch to support `pantheon-terminal`/ElementaryOS by the end of this weekend. Thanks for your help!

**Comment by Poutas** [comment]
When i enter `ctrl + shift + t` or `ctrl + shift + alt + t`,they are only open a terminal window like this 
![image](https://cloud.githubusercontent.com/assets/16750352/22535051/55e4a5ca-e933-11e6-91be-a98378a3225a.png)

**Comment by twolfson** [comment]
Right, it's currently recognizing the ElementaryOS window manager as Gnome and assuming `gnome-terminal` is the preferred terminal:

https://github.com/wbond/sublime_terminal/blob/1.16.2/Terminal.py#L99-L100

We will have to update it to identify the `--session=pantheon` flag to launch `pantheon-terminal` instead:

```
/usr/lib/gnome-session/gnome-session-binary --session=pantheon
```

**Comment by twolfson** [comment]
We have opened PR #160 to repair this issue

**Comment by twolfson** [comment]
Sorry for the delay on this. We have finally released a formal patch for this in `1.18.0`
